darshan-util/pydarshan/tests/test_summary.py

```diff
@@ -151,6 +158,11 @@ def test_main_all_logs_repo_files(tmpdir, log_repo_files):
                         # check that help message is present
                         assert "Heat map is not available for this job" in report_str
 
+                    # check if I/O cost figure is present
+                    for mod in report.modules:
+                        if mod in ["POSIX", "MPI-IO", "STDIO"]:
+                            assert "I/O Cost" in report_str
```

Comment:
If all 3 modules are present for example, you'll end up doing redundant checks here and above--you only need to check if this string is present once per report, since it only shows up once per report, regardless of the number of matching modules.